client/reader/search-stream/with-sites.jsx

```diff
@@ -164,7 +166,7 @@ class SearchStream extends React.Component {
 					</div> }
 				{ ! wideDisplay &&
 					<div className="search-stream__single-column-results">
-						{ ( searchType === POSTS && <PostResults { ...this.props } /> ) ||
+						{ ( ( searchType === POSTS || ! query ) && <PostResults { ...this.props } /> ) ||
```

Comment:
sort of. that line is confusing because of the way we are currently implicitly sneaking recs in to the stream.  

There was a bug that would have never occurred before this pr where you where if do a site search in small width, then cleared out the query, the url now retains the selection as a query arg. The old logic for inferring recs broke because of that.

If we delve back into this code later it'll be worth making isRecommendations explicit from the controllers downwards